### ISSUE TITLE
[spring-boot] Missing Chapter

### ISSUE THREAD

**Issue by cmoulliard** [fabric8-launch/appdev-documentation#812]
A concept chapter should follow the Spring Boot Preface chapter

```
Concepts

The design of a modern application designed around Spring Boot [1], embedding the Apache Tomcat Web container [2],
allows to develop Web or Websocket applications using either a HTTP Connector or non-blocking HTTP connector to reduce threads consumption. 

While such application can be packaged and deployed without specific modifications on the OpenShift platform, we invite you to look to the following Cloud - native features that we propose such as Circuit Breaker, Health Check, Server Discovery and Load Balancing, failover in order to transform your application as Cloud Microservices and take full advantage the platform capabilities. 

The different features presented hereafter are also available as a collection of Mission/Quickstart that you can use from the “Launch” Web site.

Feature | Problem addressed | Native | Framework
-- | -- | -- | --
Circuit Breaker (link to pattern) | Switch between services and continue to operate incoming request without interruption | No | Spring Cloud Kubernetes - Hystrix
Health Check | Check readiness and liveness of the service based on info exposed by spring Boot Actuator. Service will be automatically restarted if probing fails | Yes | Spring Boot Actuator
Service Discovery | Discover Service/Endpoint deployed on OpenShift and exposed behind a service or route using the service name matching a DNS entry | Yes using Kubernetes Api | -
Server Side Load balancing | Multiple instances of a service deployed will be managed by the Kubernetes Load balancer according to round-robin fashion | Yes using internal Kubernetes Load Balancer | -
Client Side Load balancing | Discover services at the client side using the Server Side Load Balancing feature | No | Spring Cloud Kubernetes - Ribbon
Externalize Parameters | Externalize application parameters to make the application independent of the environment where it runs | Yes - Kubernetes Config Map or Secret | Spring cloud Kubernetes - ConfigMap

[1] Link to spring Boot Doc
[2] Specifications table

Feature | Version
-- | --
Servlet | 3.1
JSP | 2.3
EL | 3.0
Websocket | 1.1
HTTP/2 protocol |  
```

**Comment by lavernw1** [comment]
@cmoulliard The frameworks table and its introductory statement(s).  has already been added to the SB guide.  We use a standard section for runtime details for all the RT guides and dont have a preface section. I've summarized the information on the runtime details for this section and  provided updates in the related JIRA ticket.  Typically most version related information is kept in release notes.

**Comment by tradej** [comment]
Closed via #856.
